introduction.md

```diff
@@ -4,4 +4,12 @@ Hello, My name is Tiji Mathew. I am excited to learn about Github after having c
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?